itests/kit/ensemble.go

```diff
@@ -374,17 +374,20 @@ func (n *Ensemble) Start() *Ensemble {
 
 				// we get the proof type for the requested sector size, for
 				// the current network version.
-				nv, err := m.FullNode.FullNode.StateNetworkVersion(ctx, types.EmptyTSK)
-				require.NoError(n.t, err)
-
-				proofType, err := miner.SealProofTypeFromSectorSize(m.options.sectorSize, nv)
-				require.NoError(n.t, err)
-
-				params, aerr := actors.SerializeParams(&power2.CreateMinerParams{
-					Owner:         m.OwnerKey.Address,
-					Worker:        m.OwnerKey.Address,
-					SealProofType: proofType,
-					Peer:          abi.PeerID(m.Libp2p.PeerID),
+				// nv, err := m.FullNode.FullNode.StateNetworkVersion(ctx, types.EmptyTSK)
+				// require.NoError(n.t, err)
+
+				// TODO this doesn't currently work with the FVM -- we need to specify policy somehow
+				// proofType, err := miner.WindowPoStProofTypeFromSectorSize(m.options.sectorSize)
+				// require.NoError(n.t, err)
+				// so do this instead, which works:
+				proofType := abi.RegisteredPoStProof_StackedDrgWindow64GiBV1
```

Comment:
todo

Reply:
will be fixed once we have test bundles to load.